Source/WTF/Configurations/module.modulemap

```diff
@@ -0,0 +1,62 @@
+module wtf {
+    // AutodrainedPool must never be parsed in an objective-C
+    // context.
+    explicit module AutodrainedPool {
+        requires cplusplus11, !objc
+        header "AutodrainedPool.h"
+        export *
+    }
+    // This header depends on surrounding preprocessor context.
+    // Works only on some platforms.
+    explicit module MetadataSPI {
+        requires cplusplus11
+        textual header "spi/mac/MetadataSPI.h"
+        export *
+    }
+    // The following WTF headers can be used in C contexts as well as C++
+    // so we put them in a separate submodule. We'll call it Platform
+    // since the majority of the files are Platform-related.
+    // (Some are interdependent so we couldn't split them into
+    // totally standalone submodules even if we wanted to.)
+    explicit module Platform {
+        header "Assertions.h"
```

Comment:
These headers are all "fused" and should be split into submodule-per-header

Reply:
As above, this is a pre-existing problem with the current autogenerated modulemap, so I'll fix it in a separate PR. I seem to remember that putting each `Platform*` header into its own submodule required implausible number of WebKit `#include` changes so that's why they're in one, but the others can be split off easily.